tests/test_exit_codes.py

```diff
@@ -6,7 +6,7 @@
 def test_exit_code_demo():
     """Ensure that linting the demo returns an exit code of 1."""
     try:
-        subprocess.check_output(["proselint", "--demo"])
+        subprocess.check_output(["python", "-m", "proselint", "--demo"])
```

Comment:
I'm familiar with the `-m` flag, but can you say something about what's changed that requires proselint to be run in this way?

Reply:
It isn't a requirement, so much of a better way to run tests in a sandboxed environment. For development testing, people often may want to keep the package sandboxed, and therefore not install it to their global `site-packages` - this allows them to do that and still run the tests successfully.